jekyll/_cci2/runner-server.md

```diff
@@ -1,70 +1,24 @@
 ---
 layout: classic-docs
-title: CircleCI Runner in Server
-short-title: CircleCI Runner in Server
+title: Server launch-agent compatibility
+short-title: Server launch-agent compatbilitiy
 categories: [platforms]
-description: How to set up a Runner in Server
+description: Launch-agent compatibility table for server
```

Comment:
@stalmy @nathanfishcircleci Is this how we should word/market this? I'm a bit out of the loop on the proper messaging/wording, I know it was kind of a mess with Dynamic Config / Setup Workflows so I want to be careful here.

@sarahtejas

Reply:
maybe I'm overreacting but it seems like a big change, renaming this to launch-agent. also, launch-agent as a feature name is not very intuitive. IMO from a newcomer of CircleCI etc this confuses me

Reply:
I agree, @annabaker, maybe using "Runner" here instead of "launch-agent" makes it straightforward and clear to the customer.  We can push the Runner/launch-agent pieces into the installation page

Reply:
What is the overall flow?  Will Server customers start with the Server section (independent of Runner), bounce to this page, and then... it feels as if we should give them the next step, in the form of a link to the Runner installation page.

Reply:
Sorry this wasn't completely clear: The flow is that _everyone_ starts on the "Installing Runner" page (note that it has been tagged with Cloud and Server 3.x now) and this page just holds the table of launch agents version compatibility. This is linked to from the general installation docs.